README.md

```diff
@@ -150,8 +150,8 @@ the implementation.
   - OKEx: <https://www.okex.com/join/2412712?src=from:ios-share>
   - Kucoin: <https://www.kucoin.com/ucenter/signup?rcode=r3KX2D4>
 
-  This project is maintained and supported by a small group of team. If you would like to support this project, please
-  register on the exchanges using the provided links with referral codes above.
+  This project is maintained and supported by a small group of people. If you would like to support this project, please
+  Register on the exchanges using the provided links with the referral codes above.
```

Comment:
Why change this to capital R?

Reply:
I guess he replace the list char from `-` to `+` and this is the first letter of a sentence